PowerDNS-Admin 0.4.0 - Fails pip install requirements on Python 3.11 due to ancient lxml

0.3.0 working fine but 0.4.0 seems to pull in an ancient version of lxml which is not compatible with Python 3.11.

```
#8 59.13       building 'lxml.etree' extension
#8 59.13       creating build/temp.linux-x86_64-cpython-311
#8 59.13       creating build/temp.linux-x86_64-cpython-311/src
#8 59.13       creating build/temp.linux-x86_64-cpython-311/src/lxml
#8 59.13       gcc -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -DCYTHON_CLINE_IN_TRACEBACK=0 -I/usr/include/libxml2 -Isrc -Isrc/lxml/includes -I/app/.venv/include -I/usr/include/python3.11 -c src/lxml/etree.c -o build/temp.linux-x86_64-cpython-311/src/lxml/etree.o -w
#8 59.13       src/lxml/etree.c: In function '__Pyx_PyErr_GetTopmostException':
#8 59.13       src/lxml/etree.c:261877:21: error: '_PyErr_StackItem' {aka 'struct _err_stackitem'} has no member named 'exc_type'
#8 59.13       261877 |     while ((exc_info->exc_type == NULL || exc_info->exc_type == Py_None) &&
#8 59.13              |                     ^~
#8 59.13       src/lxml/etree.c:261877:51: error: '_PyErr_StackItem' {aka 'struct _err_stackitem'} has no member named 'exc_type'
#8 59.13       261877 |     while ((exc_info->exc_type == NULL || exc_info->exc_type == Py_None) &&
#8 59.13              |                                                   ^~
#8 59.13       src/lxml/etree.c: In function '__Pyx__ExceptionSave':
#8 59.13       src/lxml/etree.c:261891:21: error: '_PyErr_StackItem' {aka 'struct _err_stackitem'} has no member named 'exc_type'
#8 59.13       261891 |     *type = exc_info->exc_type;
```

An easy way to reproduce this is change the docker version "3.17" to "edge" in docker/Dockerfile and try build, it replicates the same issue.

--- a/requirements.txt
+++ b/requirements.txt
@@ -25,7 +25,7 @@ gunicorn==20.1.0
 itsdangerous==2.1.2
 jsonschema[format]>=2.5.1,<4.0.0 # until https://github.com/Yelp/bravado-core/pull/385
 lima==0.5
-lxml==4.6.5
+--use-feature=no-binary-enable-wheel-cache lxml==4.9.0
 mysqlclient==2.0.1
 passlib==1.7.4
 #pyOpenSSL==22.1.0
@@ -33,7 +33,7 @@ pyasn1==0.4.8
 pyotp==2.8.0
 pytest==7.2.1
 python-ldap==3.4.3
-python3-saml==1.14.0
+python3-saml==1.15.0
 pytimeparse==1.1.8
 pytz==2022.7.1
 qrcode==7.3.1

